modules/efk-logging-fluentd-external.adoc

```diff
@@ -77,29 +87,19 @@ not a path on your host system.
 ====
 +
 When configuring the external aggregator, it must be able to accept messages
-securely from Fluentd.
-
-. If the external aggregator is another Fluentd server, it must have the
-`fluent-plugin-secure-forward` plug-in installed and make use of the input
-plug-in it provides:
+securely from Fluentd and must have the `fluent-plugin-secure-forward` plug-in 
```

Comment:
securely from Fluentd.  If using Fluentd 1.0 or later, configure the built-in `in_forward` plug-in with the appropriate security parameters.  If using Fluentd 0.12 or earlier, you must have the `fluent-plugin-secure-forward` plug-in installed and make use of the input plug-in it provides.  Note that the `in_forward` plug-in configuration is different than the `secure_forward` plug-in configuration.
The following configuration assumes the use of Fluentd 1.0:

Reply:
Another difference - in Fluentd 0.12, the same `fluent-plugin-secure-forward` plugin implements both the client (sending) side and the server (receiving) side.  In Fluentd 1.0 and later, `in_forward` implements the server (receiving) side, and `out_forward` implements the client (sending) side.